tests/unit/test_paths.py

```diff
@@ -200,18 +598,11 @@ def test_unlink_existing_file():
 def test_unlink_non_existing_file():
     ws = create_autospec(WorkspaceClient)
     workspace_path = WorkspacePath(ws, "/test/path")
-    ws.workspace.get_status.side_effect = NotFound(...)
+    ws.workspace.get_status.side_effect = NotFound("Simulated NotFound")
     with pytest.raises(FileNotFoundError):
         workspace_path.unlink()
 
 
-def test_relative_to():
```

Comment:
why did you remove existing test?

Reply:
There's another version of the same test that covers this (and more situations).

I've moved it here so that this is clearer from the PR diff what the change here really is.